### PR TITLE
Add unit tests to connectivity plugin

### PR DESCRIPTION
**Pull request by collinjackson**
Fixes flutter/flutter#27643
Required for merging #895

### CODE REVIEW

File: packages/connectivity/test/connectivity_test.dart [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:flutter/services.dart';
+import 'package:connectivity/connectivity.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('$Connectivity', () {
```
Comment (amirh) [inline packages/connectivity/test/connectivity_test.dart]: nit: is the `$` here on purpose?

File: packages/connectivity/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 0.4.1
+
+* Add unit tests
```
Comment (amirh) [inline packages/connectivity/CHANGELOG.md]: uber nit: period.

File: packages/connectivity/test/connectivity_test.dart (hunk 1 above)
Comment (collinjackson) [reply packages/connectivity/test/connectivity_test.dart]: Yeah, the idea is to force you to update the string if you change the class name

Comment (amirh) [reply packages/connectivity/test/connectivity_test.dart]: TIL! nice trick

**Comment by amirh** [comment]
Also don't forget to make the formatter happy before merging

**Pull request merged**